README.md

```diff
@@ -111,6 +110,7 @@ Deprecated events:
 
 * `contentupdate` (EVENT) -- Replaced by `contentchanged`, which is more reliable.
 * `adscanceled` (EVENT) -- Intended to cancel all ads, it was never fully implemented. Instead, use `nopreroll` and `nopostroll`.
+* `adserror` (EVENT) -- This event skips prerolls when seen before a preroll ad break. It skips postrolls if called after contentended and before a postroll ad break. It ends linear ad mode if seen during an ad break. It can be replaced using `skipLinearAdMode` and `endLinearAdMode`.
```

Comment:
I don't think adserror was actually deprecated

Reply:
I'm deprecating it in this PR.